src/components/widgets/camera/CameraItem.vue

```diff
@@ -105,12 +105,12 @@ export default class CameraItem extends Vue {
     const config = this.camera
     let transforms = ''
 
-    if (!config.rotate) {
+    if (config.rotation === 0) {
```

Comment:
`rotation` can be `undefined` too (legacy cameras)
```suggestion
    if (!config.rotation) {
```

Reply:
I made sure that doesn't happen here: https://github.com/fluidd-core/fluidd/pull/839/files#diff-11df4c839902d11995d754a99da6bcd47e702247233ccecaf07a8b455628b14aR46

Reply:
Ah! Odd, one of my cameras still had it as undefined, but I just tried it again and it worked fine :)